### PR TITLE
Namedtuples for dielectric 

### LINKED ISSUE #539: Return namedtuples in dielectric.py functions
There are a few functions in [`plasmapy/physics/dielectric.py`](https://github.com/PlasmaPy/PlasmaPy/blob/1511167bfb7631b0b54d1e4919d0adaf001c59aa/plasmapy/physics/dielectric.py) that return three values, e.g.
```Python
def cold_plasma_permittivity_SDP(B, species, n, omega):
    ...
    return S, D, P
```
It would probably be helpful if we could return these as `collections.namedtuple` objects (or perhaps a new class) since that would group the values together in a convenient object.  If it is returned as a `namedtuple`, then it should preserve existing behavior when doing: 
```Python
>>> S, D, P = cold_plasma_permittivity_SDP(B, species, n, omega)
```
Thanks!
-Nick

### PR DESCRIPTION
**Pull request by StanczakDominik**
Closes #539. Names can of course probably be improved.

### CODE REVIEW

File: plasmapy/physics/dielectric.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
            'cold_plasma_permittivity_LRP',
            'permittivity_1D_Maxwellian']
 
+StixTensorElements = namedtuple("Result1", ["sum", "difference", "plasma"], )
+RotatingTensorElements = namedtuple("Result1", ["left", "right", "plasma"], )
```
Comment (namurphy) [inline plasmapy/physics/dielectric.py]: Would this perhaps be more descriptive?
```Python
StixTensorElements = namedtuple("StixTensorElements", ["sum", "difference", "plasma"], )
RotatingTensorElements = namedtuple("RotatingTensorElements", ["left", "right", "plasma"], )
```

Comment (StanczakDominik) [reply plasmapy/physics/dielectric.py]: Gah! Of course I would forget to change those.

Comment (StanczakDominik) [reply plasmapy/physics/dielectric.py]: Addressed by afa8919

File: plasmapy/physics/tests/test_dielectric.py [hunk 2]
```diff
@@ -46,12 +48,23 @@ def test_proton_electron_plasma(self):
         P_analytical = 1 - (omega_pe ** 2 + omega_pp ** 2) / omega ** 2
 
         species = ['e', 'p']
-        S, D, P = cold_plasma_permittivity_SDP(B, species, n, omega)
+        S, D, P = tuple_result = cold_plasma_permittivity_SDP(B, species, n, omega)
```
Comment (namurphy) [inline plasmapy/physics/tests/test_dielectric.py]: Whoa!  I forgot that we could do chained assignments.  Makes me appreciate Python all over again. :cat:

**Comment by StanczakDominik** [comment]
Hey @namurphy, I think this could be merged now. :)

**Comment by StanczakDominik** [comment]
Darn, I missed the comment you made about the docstrings. Apologies. Fixing those now.

**Pull request merged**